src/routes/routesConfig.tsx

```diff
@@ -16,18 +16,30 @@
 
 import React, { type ReactNode } from 'react';
 
-import HomePage from 'src/content/home/HomePage';
-import SpeciesSelectorPage from 'src/content/app/species-selector/SpeciesSelectorPage';
+import HomePage, {
+  serverFetch as homePageServerFetch
+} from 'src/content/home/HomePage';
+import SpeciesSelectorPage, {
+  serverFetch as speciesSelectorServerFetch
+} from 'src/content/app/species-selector/SpeciesSelectorPage';
 import SpeciesPage, {
   serverFetch as speciesPageServerFetch
 } from 'src/content/app/species/SpeciesPage';
-import GenomeBrowserPage from 'src/content/app/genome-browser/BrowserPage';
+import GenomeBrowserPage, {
+  serverFetch as genomeBrowserPageServerFetch
+} from 'src/content/app/genome-browser/BrowserPage';
 import EntityViewerPage, {
   serverFetch as entityViewerServerFetch
 } from 'src/content/app/entity-viewer/EntityViewerPage';
-import BlastPage from 'src/content/app/tools/blast/BlastPage';
-import AboutPage from 'src/content/app/about/AboutPage';
-import HelpPage from 'src/content/app/help/HelpPage';
+import BlastPage, {
+  serverFetch as blastServerFetch
```

Comment:
Do we need to keep the server fetch names consistent? Entity viewer and Blast don't have `page` in their imported names.